src/ui/react/src/uibridge/richcombo.jsx

```diff
@@ -219,7 +219,20 @@
          */
         init: function(editor) {
             editor.ui.addRichCombo = function(richComboName, richComboDefinition) {
-               this.add(richComboName, CKEDITOR.UI_RICHCOMBO, richComboDefinition);
+                this.add(richComboName, CKEDITOR.UI_RICHCOMBO, richComboDefinition);
+
+                var e = new Error();
+                // In IE 9 - IE 11, e = {number: 0, description: ''};
+                // so check the "stak" property
+                if (!e.stack) {
+                    return;
+                }
+
+                var pluginName = /plugins\/(.*)\/plugin.js/.exec(e.stack)[1];
```

Comment:
Hey @ipeychev, what do you think about this? 

It is the only way we could figure to actually map the plugin names. This will happen rarely, so we can take the performance hit. The only problem I see is that in a bundle scenario, this might not work... In light of that, we may not be able to implement this in a safe and consistent way... what do you think?